Proposed changes to Bylaws and Policies - September 2025 (#3)
ENTS-Source/bylaws-policies: bylaws/bylaws.md

    2. Composition

>         1. The Board shall consist of three to seven members. The Board is elected by the members of The Society.

Author: ```suggestion
        1. The Board shall consist of five members. The Board is elected by the members of The Society.
```

    6. Directors Meetings

>         1. Where the Board is not performing its duties and responsibilities monthly, the Board shall meet at least monthly.

Author: ```suggestion
        1. The Board shall meet at least monthly.
```

        2. The Board may set regulations governing the calling and medium of its meetings (notice, date, time, and place), the conduct of business thereat, and generally as to the conduct of its affairs.  These will be documented in The Society Policies.
        3. The President shall call and chair the meetings.  The President shall also call a meeting if any two Directors make such a request by providing notice or while having the floor at a general meeting or Board meeting, and state the business for the meeting.

>         4. Three days notice for regular Directors Meetings is provided to each Director.  As much notice as is possible shall be given for unscheduled or urgent meetings.

Author: ```suggestion

```

and renumber

        6. One half of Directors, rounded up, shall be considered a quorum for all Directors Meetings; votes shall be passed by a simple majority unless otherwise agreed upon.
        7. If there is no quorum, the chairperson may adjourn the meeting to the same time, place, and day of the following week.  If urgent matters requiring quorum have been tabled as a result, The Directors present at this later meeting shall constitute quorum regardless of their number.
        8. Each Director has one vote.
        9. Directors Meetings are open to Members in Good Standing, but only Directors may vote.  A majority vote of The Board may ask any other Member or other persons to leave.
        10. A majority vote of The Board may move a meeting, or any portion thereof, in-camera at any time.
        11. All Directors may agree to and sign a resolution at any time.  This resolution is as valid as one passed at any Directors Meeting.  It is not necessary to give notice or to call a Directors Meeting for the resolution in this case.  The date on the resolution is the date it is passed.

>         12. Any member of The Board unable to attend a meeting may send a proxy to represent them at that meeting.  The proxy shall:

Author: ```suggestion
        12. Directors shall not be able to assign proxies to represent them at Directors Meetings.
```
